core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/UnsetClause.kt

```diff
@@ -1,31 +1,32 @@
 package ch.ergon.dope.resolvable.clause.model
 
 import ch.ergon.dope.DopeQuery
-import ch.ergon.dope.resolvable.clause.IUpdateSetClause
 import ch.ergon.dope.resolvable.clause.IUpdateUnsetClause
 import ch.ergon.dope.resolvable.expression.unaliased.type.Field
 import ch.ergon.dope.resolvable.formatToQueryString
 import ch.ergon.dope.validtype.ValidType
 
 class UnsetClause(
-    private val field: Field<out ValidType>,
-    private vararg val fields: Field<out ValidType>,
-    private val parentClause: IUpdateSetClause,
+    field: Field<out ValidType>,
+    private val fields: MutableList<Field<out ValidType>> = mutableListOf(),
```

Comment:
same comment as in the SetClause: Can't you use vararg instead of a list like in the other clauses?

Reply:
done